### DISCUSSION TITLE
How to open the login entries search window from the UI?

### DISCUSSION THREAD

**Discussion by MurzNN** [quexten/goldwarden#75] [Q&A]
I can open the login entries search window by the autotype shortcut and it works. But how can I open it just via the mouse using the main window UI?

I tried to click and double-click on the "Vault Login Entries" section but nothing happens, and I see no suitable buttons for this.

**Comment by quexten** [comment]
![image](https://github.com/quexten/goldwarden/assets/11866552/9cfd4572-f017-46a9-af3d-d2997f336976)
Top button under actions - "Autotype"

**Reply by MurzNN** [reply]
Hmm, but this is Autotype, not the search to copy-paste. So maybe rename this button to "Search"?
Cuz, as I understand, the UI button can't be used for the autotype feature, because w can't choose the external window where to autotype, right?

**Reply by quexten** [reply]
Yeah, I agree, there is another related discussion here https://github.com/quexten/goldwarden/discussions/46

There could be even other features added like totp code, but also generating passwords etc.

**Reply by quexten** [reply]
>  Cuz, as I understand, the UI button can't be used for the autotype feature, because w can't choose the external window where to autotype, right?

Well, right now the auto-type immediately happens on the window next in focus. This usually works on floating window managers, but on tiling managers this currently still causes problems. Then again, I don't know of any tiling managers that implement the remote desktop portal required for auto-typing so it only currently affects kde/gnome with tiling plugins (I think?)

**Reply by quexten** [reply]
https://github.com/quexten/goldwarden/issues/76
